stumpy/stumped.py

```diff
@@ -54,13 +56,17 @@ def stumped(dask_client, T_A, m, T_B=None, ignore_trivial=True, normalize=True,
         The p-norm to apply for computing the Minkowski distance. This parameter is
         ignored when `normalize == True`.
 
+    k : int
+        The number of smallest elements in distance profile that should be stored
+        for constructing the top-k matrix profile.
+
     Returns
     -------
     out : numpy.ndarray
-        The first column consists of the matrix profile, the second column
-        consists of the matrix profile indices, the third column consists of
-        the left matrix profile indices, and the fourth column consists of
-        the right matrix profile indices.
+        The first k columns consist of the top-k matrix profile, the next k columns
```

Comment:
See proposed docstring above

Reply:
Sure. I will provide explanation for `k=1` and `k>1`